doc/debug-sections.txt

```diff
@@ -57,14 +60,14 @@ section 22    Refresh Calculation
 section 23    URL Parsing
 section 23    URL Scheme parsing
 section 24    SBuf
-section 25    MiME Header Parsing
 section 25    MIME Parsing and Internal Icons
+section 25    MiME Header Parsing
```

Comment:
Here and elsewhere, we should polish the source code comments (that inject these lines) to use consistent spelling/case, but, again, polishing of individual section comments should stay outside this PR scope IMO.